src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int COLOR_COUNT = Colors.values().length;
+    private final Random random = new Random();
+
     public String getRandomColor() {
-        return null;
+        int colorNumber = random.nextInt(COLOR_COUNT);
+        return "" + Colors.values()[colorNumber];
```

Comment:
[Use name() for getting String representation of enum constants](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#use-name-for-getting-string-representation-of-enum-constants)
(`"" + Colors.values()[colorNumber]` here `toString` will be called automatically)